src/data/faq.json

```diff
@@ -2171,66 +2171,304 @@
-                    "description": "Abbreviation of \"Digital COVID Certificate\"."
+                    "description": "Abbreviation for the EU Digital COVID Certificate. The certificate was developed with the aim of facilitating the free movement of persons in the EU."
+                },
+                {
+                    "term": "Diagnosis Key",
+                    "anchor": "diagnosis_key",
+                    "description": "See <a href='#glossary_temporary_exposure_keys'>Temporary Exposure Keys</a>"
                 },
                 {
                     "term": "Digital Certificate",
                     "anchor": "digital_certificate",
-                    "description": "A certified document. It contains the subject's health information signed by a trusted issuer. <br/><br/> Every digital certificate has an issue date and an expiration date. After the certificate expires it must be renewed. <br/><br/> In the CWA we differentiate between: <br/><ul><li>Vaccination Certificate</li><li>Test Certificate</li><li>Recovery Certificate</li></ul>"
+                    "description": "A digital certificate is an authenticated digital document. It contains at least the holder's name, the holder's public key, and the digital signature (<a href='#glossary_signature'>signature</a>) of the issuing entity. <br/><br/> Each digital certificate has an issue date and a time-limited <a href='#glossary_validity'>validity</a> (expiration date). After the validity expires, the certificate must be renewed. <br/><br/> In the CWA, we distinguish between: <br/><ul><li>Vaccination certificate</li><li>Test certificate</li><li>Recovery certificate</li></ul>"
                 },
                 {
                     "term": "Digital Signature",
                     "anchor": "digital_signature",
                     "description": "See <a href='#glossary_signature'>Signature</a>"
+                },
+                {
+                    "term": "Digital Vaccination Certificate",
+                    "anchor": "digital_vaccination_certificate",
+                    "description": "See <a href='#glossary_digital_certificate'>Digital Certificate</a>"
                 }
             ],
             "E": [
+                {
+                    "term": "Effective validity",
+                    "anchor": "effective_validity",
+                    "description": "We refer to the duration of vaccination protection as 'effective validity'. The effective validity of the vaccination thus counts from the last vaccination date. While in Germany the professional validity is currently set at one year, the validity of the vaccination protection in other countries may differ from this. In the case of a <a href='#glossary_recovery_certificate'>recovery certificate</a>, the effective validity is set at 'date of the person's first positive <a href='#glossary_pcr_test'>PCR test</a> result' plus 180 days, according to the EU eHealth Network."

```

Comment:
To be consistent, "professional validity" should be called "effective validity".

Reply:
Will be corrected in the next commit, thanks!